src/Mutant/TestFrameworkMutantExecutionResultFactory.php

```diff
@@ -77,6 +77,7 @@ public function createFromProcess(MutantProcess $mutantProcess): MutantExecution
             $mutant->getPrettyPrintedOriginalCode(),
             $mutant->getMutatedCode(),
             $mutant->getTests(),
+            $mutantProcess->getFinishedAt() - $process->getStartTime(),
```

Comment:
does it make sense to have the same name convention?

- `getStartedAt()`
- `getFinishedAt()`